Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
```

Comment:
Why have a `Gemfile` at all in this project? I thought `Gemfile` was only for repos that are going to be deployed somewhere (as opposed to being required by other repos that are themselves deployed).